.github/workflows/ci.yml

```diff
@@ -32,7 +44,11 @@ jobs:
           - "7.0"
           - "8.0"
         exclude:
-          - container_image: docker.io/library/alpine:latest
+          container_image: 
+            - docker.io/library/alpine:latest 
+            - docker.io/arm64v8/alpine:latest 
+            - docker.io/s390x/alpine:latest 
+            - docker.io/ppc64le/alpine:latest
```

Comment:
I find this yaml confusing, but I assume it means we're skipping the 8.0 on these Alpine images? Why is that?
If we're skipping the 8.0, maybe we should just leave these images out completely?

Reply:
I set arm64, s390x and ppc64le to skip 8.0 because docker.io/library/alpine:latest had been already set to skip it for reasons presently unknown to me. I assumed there would be a similar issue with the other three architectures. I don't mind leaving out arm64, s390x and ppc64le for alpine for now.